geoips/interfaces/base.py

```diff
@@ -670,16 +714,16 @@ def get_plugin(self, name):
         # Find the plugin module
         try:
             if exists(name):
+                # This is used! For output checkers at least.
```

Comment:
https://github.com/NRLMMD-GEOIPS/geoips/pull/419#issuecomment-1924471870

Can we ignore this for now?  Looks like we need to open 2 issues, then this is good to go?  Look into if we can remove this (I'm guessing no, since I *just* added it...), and look into whether get_plugin can be combined (probably, since we have officially moved to a single source/product for for product plugins). @jsolbrig

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/435

Reply:
@mindyls Was there some reason this was different in the open-source GeoIPS?  I'm sure we got rid of the output_checkers special case internally.

Reply:
I expect the change was just made after 1.12.0 was bundled up for release.  I say resolve this, and it will be gone in the next release.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/419#issuecomment-1927934980
There is lance's original comment RE: getting rid of that special case, for reference.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/419#issuecomment-1924471870 <-- Jeremy's original comment, for reference.  Hidden as duplicate now.

Reply:
Okay, sounds good. I created #440 to consider combining get_plugin on `BaseInterface`.